hack/build/run-tests.sh

```diff
@@ -22,4 +22,10 @@ source hack/build/common.sh
 # parsetTestOpts sets 'pkgs' and test_args
 parseTestOpts "${@}"
 
-go test -v -test.timeout 90m ${pkgs} ${test_args:+-args $test_args}
+test_command="go test -v -test.timeout 90m ${pkgs} ${test_args:+-args $test_args}"
```

Comment:
90m should be changed to something like 15m for the CDI tests. This lets us timeout quicker if  something hangs.  The 90m is left over from kubevirt's functional test suite taking forever to run now.

Reply:
How much of that time is env setup though?  The func tests run in 2 minutes on my laptop after the cluster has been set up.

Reply:
That is reported test time by the junit reporter. An entire run is over 30 minutes including setup. See http://jenkins.ovirt.org/job/kubevirt_containerized-data-importer_standard-check-pr/50/testReport/ for the last run.